app/views/help.scala.html

```diff
@@ -308,32 +318,83 @@ <h3 class="govuk-heading-s" id="file-checks">Step four: File checks</h3>
+            <p class="govuk-body">You may now add, edit or delete descriptive and closure metadata.</p>
+            <h3 class="govuk-heading-s" id="descriptive-metadata">Descriptive metadata</h3>
+            <p class="govuk-body">You are not required to add descriptive metadata to your records, but it can enhance them.</p>
+            <p class="govuk-body">Once you select the file or files you wish to add descriptive metadata to, you may provide the following information:</p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>Description – this will be visible to the public to help explain the content of your file or files</li>
+                <li>Date of record – this is prepopulated and shows the date last modified</li>
+                <li>Language</li>
+            </ul>
+
+            <h3 class="govuk-heading-s" id="closure-metadata">Closure metadata</h3>
+            <p class="govuk-body">If you wish to transfer closed files and folders, you must add closure metadata.</p>
+            <p class="govuk-body">
+                Once you select the file or files to add closure metadata to, you must first confirm the closure status has been approved by the Advisory Council.
+                If it has been approved, you may proceed to add the following information:
+            </p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>FOI decision asserted – this is the date of the Advisory Council approval</li>
+                <li>Closure start date – this is the date of the last meaningful change to the record</li>
+                <li>Closure period – this is the number of years the record is closed from the closure start date</li>
+                <li>FOI exemption code – you should search and select at least one exemption code per file</li>
+                <li>Title sensitivity – you should confirm if the title is sensitive to the public. If it is, you can add an alternative title</li>
+                <li>Description Sensitivity – you can only provide an alternative description if you have set a Descriptive Metadata page first</li>
+            </ul>
+
+            <h3 class="govuk-heading-s">Titles and descriptions that are sensitive to the public</h3>
+            <p class="govuk-body">
+                If the title or description of your record is sensitive to the public, you can provide an alternative when adding closure metadata.
+                The alternative will be visible on the public catalogue. You may wish to do one of the following:
+            </p>
+            <p class="govuk-body">
+                Redact – remove the sensitive words. For example: ‘Correspondence with [name withheld]’
+            </p>
+            <p class="govuk-body">
+                Sanitise – replace the sensitive word. For example: ‘UK military exercises in xx’ (where military has replaced Special Forces).’
```

Comment:
Use of single quotes don't make sense. Should be like this: `... For example: 'UK military exercises in xxx' (where 'military' has replaced 'Special Forces').`